backend/sirius-web-collaborative-diagrams-api/src/main/java/org/eclipse/sirius/web/collaborative/diagrams/api/IDiagramService.java

```diff
@@ -27,6 +27,8 @@
 public interface IDiagramService {
     Diagram create(DiagramCreationParameters parameters);
 
+    Diagram create(DiagramCreationParameters parameters, Diagram previousDiagram);
```

Comment:
This seems... strange. We ask for the _creation_ (not update or refresh) of a diagram, but give an already `previousDiagram` as a parameter? Maybe it's juste the method and/or parameter name that should be different/more explicit, but in terms of API its confusing.

Reply:
I have rewrote that